### ISSUE TITLE
PIDs with more than 5 digits

### ISSUE THREAD

**Issue by Baschdl** [Syllo/nvtop#67]
nvtop doesn't show PIDs with more than 5 digits:
![image](https://user-images.githubusercontent.com/3034832/82252099-aa301c80-994e-11ea-8551-69c32a6e1344.png)
![image](https://user-images.githubusercontent.com/3034832/82252140-bcaa5600-994e-11ea-8c13-4c7046a1ff1a.png)
Started nvtop with `docker run --runtime nvidia --rm -ti 69guitar1015/nvtop`

**Comment by gothicVI** [comment]
I'd assume changing
```
    [process_pid] = 5,       [process_user] = 4,          [process_gpu_id] = 3,
```
to
```
    [process_pid] = 6,       [process_user] = 4,          [process_gpu_id] = 3,
```
in https://github.com/Syllo/nvtop/blob/master/src/interface.c#L174 should do the trick.

**Comment by Syllo** [comment]
Thank you for the report.
Apparently it is more complicated than that: the maximum pid value can be read and set through the ```/proc/sys/kernel/pid_max``` interface, up to a maximum of 2^22 on a 64-bit system.
Hence, I'll put 7 to be safe for now!

**Comment by Syllo** [comment]
Fixed in ecc5a36e2899005bf38ece0710709f4e5950ebb0
